### PR TITLE
Interaction States - NxFileUpload - NxMultiFileUpload - RSC-1468

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-1468

### CODE REVIEW

**Comment by EmilyBermudez88** [comment]
It looks like MultipleFileUpload was automatically moved to 'Done' but its snapshots need to be updated as well. What do you think about updating them in this ticket, since the interaction states will be the same for both?

File: lib/src/components/NxFileUpload/NxFileUpload.scss [hunk 1]
```diff
@@ -20,7 +20,6 @@
     // focus styles must be reimplemented because it isn't the actual button that gets focus, it's the input
     &:focus + .nx-btn {
       border-color: var(--nx-color-interactive-border-focus);
```
Comment (EmilyBermudez88) [inline lib/src/components/NxFileUpload/NxFileUpload.scss:22]: Looks like you need to re-implement the tertiary button focus style here. Currently when you tab through, you get the blue border when focused instead.

**Comment by entroform** [comment]
> MultipleFileUpload


> It looks like MultipleFileUpload was automatically moved to 'Done' but its snapshots need to be updated as well. What do you think about updating them in this ticket, since the interaction states will be the same for both?

I thin

> It looks like MultipleFileUpload was automatically moved to 'Done' but its snapshots need to be updated as well. What do you think about updating them in this ticket, since the interaction states will be the same for both?

That's a good idea!
I've updated the snapshots here: 91140c6a30d6a1b23900753cefc8fe36ebe2c669 will note on the ticket as well

**Review by EmilyBermudez88 — APPROVED** [review]
LGTM! Just look into the comment https://github.com/sonatype/sonatype-react-shared-components/pull/1002#discussion_r1132409589

File: lib/src/components/NxFileUpload/NxFileUpload.scss [hunk 2]
```diff
@@ -103,22 +104,32 @@
...
+    outline-offset: -1px;
 
-    &:hover, &:active {
-      border-color: var(--nx-swatch-grey-05);
+    &:hover {
+      outline-color: var(--nx-color-interactive-border-hover);
     }
 
+    &:hover,
     &:active {
```
[the remark this answers is not in the document]
Comment (entroform) [reply lib/src/components/NxFileUpload/NxFileUpload.scss]: > I don't think we need the `&:active` here? Background colour in active state is dealt with below.

Ah yup that's not needed!
Thanks! a4d59fc94e2aa685906697fb0e44efddf09d57c6

**Pull request merged**